subsys/bluetooth/controller/hci_driver.c

```diff
@@ -656,7 +655,7 @@ static int configure_supported_features(void)
 
 static int configure_memory_usage(void)
 {
-	int required_memory;
+	int required_memory = 0;
```

Comment:
So we want this to return 0 when CONFIG_BT_PERIPHERAL=n && CONFIG_BT_CENTRAL=n && CONFIG_BT_LL_SOFTDEVICE_MULTIROLE=n , is that right?

Won't this fail at sdc_enable() since the defaults are non-zero?

Reply:
I removed this hunk now as its not needed with latest changes... 
I only added it because we got comipler warnings when required_memory was never written, but still read.
But with latest changes it is always written.